astrology.lic

```diff
@@ -276,13 +289,14 @@ class Astrology
     fix_standing
   end
 
-  def train_astrology
-    check_ripples
+  def train_astrology(settings)
+    return if DRSkill.getxp('Astrology') > 30
```

Comment:
Should this check only be here, before we do any training, or checked after each step of training?